README.md

```diff
@@ -61,6 +61,12 @@ To run with images and payments services:
 $ COMPOSE_PROFILES=images,payments ./sndev start
 ```
 
+To run with the option to sign up/login using [MailHog](#MailHog):
+
+```sh
+$ COMPOSE_PROFILES=email ./sndev start
+```
+
```

Comment:
This will run by default so I think adding this to the README is a bit confusing. We already suggest how to start only specific resources above.

Reply:
Yeah, makes sense. I'm going to remove it

Reply:
eef9b3de4d0c0ae3dbab9671a81737cbef108332